docs/handbook/getting-started.md

```diff
@@ -898,13 +898,6 @@ For some guidance writing this section:
 * Installation videos: [web link](https://www.youtube.com/watch?v#VVWP_5oAy3w)
 </div>
 
-<i class="fa fa-info-circle fa-lg" aria-hidden="true"></i> **NOTE:**
```

Comment:
This paragraph is removed, because it references to GRUB 0.97 menu.lst file, which really you can't find nowadays in the wild.